### PR TITLE
Restyle pages with newcss

### PR DESCRIPTION
**Pull request by jinmayamashita**
## Why

I want to make it a little easier to understand in terms of UI about the functions we offer with a minimum of css styles.

## How

Introducing [new.css](https://newcss.net/) so that styling can be done with HTML only,
Re style the pages for each function as below.

Additionally:

- Remove unnecessary files
- Refactored some components to be more concise

### CODE REVIEW

File: src/main.scss [hunk 1]
```diff
@@ -1,13 +1,47 @@
+@import url(https://fonts.xz.style/serve/inter.css);
+@import url(https://cdn.jsdelivr.net/npm/@exampledev/new.css@1.1.2/new.min.css);
```
Comment (jinmayamashita) [inline src/main.scss:2]: The newcss is being imported for the appearance of the demo pages.
When using boilerplate, you can delete the style definition here and define your own style.

File: package.json [hunk 2]
```diff
@@ -26,7 +26,6 @@
     "@playwright/test": "1.24.0",
     "@storybook/addon-actions": "6.5.9",
     "@storybook/addon-essentials": "6.5.9",
-    "@storybook/addon-links": "6.5.9",
```
Comment (jinmayamashita) [inline package.json:29]: For now, `@storybook/addon-links` feels unnecessary

File: src/main.scss (hunk 1 above)
Comment (mvn-dongho-dn) [reply src/main.scss:2]: Should we use the `<link>` element and put this to `index.html` instead of using `@import` to make it download parallelly?

Comment (jinmayamashita) [reply src/main.scss:2]: Thank you! I certainly didn't think through to performance. (That's a difference of about 150 ms :eyes:)

- when using `<link>` element (541ms)
  - スクリーンショット 2022-07-27 13 30 11
- when using `@import` (696ms)
  - スクリーンショット 2022-07-27 13 30 11

Usually whether through links or through `@import` are bad practice for high performance web sites as you said.

The reason for adding `@import` directly in the `main.scss` file is not so much for performance as it is to make it easier to customize styles when using boilerplate for the first time. (Developers need to know that they are loading in a link tag in index.html to turn off the styles 😅)

@mvn-dongho-dn What do you think about this?

Comment (mvn-dongho-dn) [reply src/main.scss:2]: Yeah, I agree with you that it makes developers easier to customize. But I am just concerned if someone they don't know about @import is bad practice and they think this is a convention of boilerplate => they still add more @import external CDN link CSS to this file 😅
So, I think we should leave the comment at the top of `main.scss` file like this:
```
// The @import CSS link here is just for Demo, you can remove it or move it to the index.html file by tag `<link>` in your project for best practice
@import url(https://fonts.xz.style/serve/inter.css);
@import url(https://cdn.jsdelivr.net/npm/@exampledev/new.css@1.1.2/new.min.css);
....
```

In addition, after I build App and run it on the browser and see different downloading.
- With `@import`
Screen Shot 2022-07-27 at 11 17 20 AM

- With `<link>`
Screen Shot 2022-07-27 at 10 47 12 AM

@jinmayamashita  What do you think about this?

Comment (jinmayamashita) [reply src/main.scss:3]: @mvn-dongho-dn 
I totally agree with your comment that the developer may continue bad practice on the use of `@import` :+1:
(+ Thanks for the additional resources!)

I have added a comment as you suggested bd94966a5e7bbbc5b13860c623f043863ac1d86c

**Comment by jinmayamashita** [comment]
@mvn-dongho-dn Sorry, can you check the PR #84? (e2e test failure issue)
If the PR #84 is merge, I will then incorporate it into this PR :pray:

**Comment by mvn-dongho-dn** [comment]
@jinmayamashita   I see a problem here
Screen Shot 2022-07-28 at 10 37 25 AM
Could you change `/api/v1/login` => `/api/login`

**Comment by jinmayamashita** [comment]
API endpoint is correct. `/api/v1/login`

スクリーンショット 2022-07-28 16 16 16

I think this issue is `msw init` related issue.
After importing the PR #84, I can run the e2e test and see that it succeeds.

スクリーンショット 2022-07-28 16 14 58

@mvn-dongho-dn Would you confirm this procedure?

**Comment by mvn-dongho-dn** [comment]
@jinmayamashita  It's okay Jinma san. I have no comment for PR #84, I just approved.

**Comment by jinmayamashita** [comment]
@mvn-dongho-dn
I　just merged #83 into this PR. please review when you have free time :bow:

**Review by mvn-dongho-dn — APPROVED** [review]
It's okay @jinmayamashita san, approved.

**Pull request merged**